.travis.yml

```diff
@@ -5,9 +5,9 @@ before_install:
   - "sh -e /etc/init.d/xvfb start"
 
 node_js:
-  - '0.10'
+  - '4'
 
 before_script:
-  - 'npm install'
+  - 'npm run lint'
```

Comment:
`npm install` is already run before this stage, so the build used to call `npm install` twice.